test/inductor/test_cutlass_backend.py

```diff
@@ -425,6 +423,8 @@ def compare_results(
                 "max_autotune_gemm_backends": max_autotune_gemm_backends,
                 "cuda.cutlass_dir": _CUTLASS_DIR,
                 "cuda.cutlass_max_profiling_configs": 4,
+                "cuda.cutlass_op_allowlist_regex": "",
+                "cuda.cutlass_op_denylist_regex": "pingpong",  # Pingpong Kernels can lead to numerical issues
```

Comment:
is there documentation anywhere of the potential ops ? where is pingpong coming from ?

Reply:
Well, that's a Cutlass internal thing how Kernels are named. New Kernels are being introduced with new Versions of Cutlass all the time. A current list could be generated by calling op.configuration_name() on all Cutlass ops, for example those returned from cutlass_utils.gen_ops(): https://github.com/pytorch/pytorch/blob/438388245c1c98607d8667a617c6af17e5438fef/torch/_inductor/codegen/cuda/cutlass_utils.py#L173  - The Kernels with "pingpong" in their name seems to use a certain zig-zag memory access pattern (thereby the name pingpong ), which in my experience often led to stronger numerical discrepancies with ATen / eager ops for larger GEMMs where many numbers are being added.